src/main/java/de/muenchen/isi/api/validation/FoerdermixNotEmptyValidator.java

```diff
@@ -0,0 +1,22 @@
+package de.muenchen.isi.api.validation;
+
+import de.muenchen.isi.api.dto.FoerdermixDto;
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import lombok.NoArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@NoArgsConstructor
+public class FoerdermixNotEmptyValidator implements ConstraintValidator<FoerdermixNotEmptyValid, FoerdermixDto> {
```

Comment:
Die Klasse Unit-Testen.

Reply:
Jede Klasse soll in ihrem eigenen Unit-Test getestet werden. Somit ist sofort ersichtlich, welcher Klasse fehlerhaft ist.

Es sind auch die Grenzfälle abzutesten.